packages/autocomplete-plugin-query-suggestions/src/__tests__/createQuerySuggestionsPlugin.test.ts

```diff
@@ -394,6 +435,59 @@ describe('createQuerySuggestionsPlugin', () => {
     });
   });
 
+  test('accumulates suggestion categories from multiple indexes', async () => {
+    castToJestMock(searchClient.search).mockReturnValueOnce(
+      Promise.resolve(
+        createMultiSearchResponse({
+          hits: multiIndexHits,
+        })
+      )
+    );
+
+    const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+      searchClient,
+      indexName: 'indexName',
+      categoryAttribute: [
+        'index_1|index_2',
```

Comment:
I haven't really thought about this feature yet, but api-wise, I think it would be better if this was an array instead of a magic string:

```suggestion
        ['index_1', 'index_2'],
```

Reply:
Probably even nested arrays so that you can target different attribute names:

```js
[
  ['index_1', 'facets', 'exact_matches', 'data_origin1'],
  ['index_2', 'facets', 'exact_matches', 'data_origin2']
]
```

Reply:
yes, looking at the example again, the whole item should be an array if we accept multiple indices

Reply:
I thought a lot about how it could work, but I didn't came out with a case where we need to use different attribute names (maybe there is a case). `data_origin1 ` and `data_origin2 ` basically need to be a single attribute — `data_origin`, but this attribute would contain different values per index. So basically it almost the same as current `categoryAttribute`, but it searches multiple indexes, and picks only the first match. 

I need to wrap my head around a multi-attribute use-case, it wouldn't hurt probably, but currently I can't come up with an example usage for this — the current PR logic would accumulate and sort the multi-index matches like they're of a same "kind", but they can be completely different with multiple attributes

Reply:
I also described this in a "Details" section of PR description. Will providing different attribute names indeed make sense?

Reply:
I see how in most/all use cases the facets would be completely identical, but I could imagine another index where the facet values are the same but translated, so they wouldn't be identical.

I prefer handling an optionally nested array over magic strings, as people could already be using | in their index names

Reply:
Yeah, I also agree with arrays, just wasn't sure about multiple attributes, will redo it shortly

Reply:
I did it, also outlined the docs in `categoryAttribute` description